blockbase/generate-children.mjs

```diff
@@ -0,0 +1,73 @@
+import { promises as fs } from 'fs';
+import deepmerge from 'deepmerge';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+const localpath = dirname( fileURLToPath( import.meta.url ) );
+
+(async function start() {
+	let args = process.argv.slice(2);
+	return await generateChildren();
+})();
+
+async function getPackageJson( directory ) {
+    const packageJsonString = await fs.readFile( directory + '/package.json', 'utf8' );
+    return JSON.parse( packageJsonString );
+}
+
+async function generateChildren() {
+	const children = [
+		{
+			"name": 'Geologist',
+			"description": "Geologist is a streamlined theme for modern bloggers. It consists of a simple single column of posts, paired with a sophisticated color palette and beautiful sans-serif typography.",
+			"slug": 'geologist'
+		},
```

Comment:
we could maybe pick this up from the stylesheet of the themes? and just give an array of the themes that are the current children

Reply:
I did consider that, but I think ultimately this script would also generate the style sheets.